src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -42,10 +45,7 @@
  */
 public class CSVImporter {
 
-  /**
-   * A {@link java.util.logging.Logger} for this class.
-   */
-  private static final transient Logger logger = Logger.getLogger(CSVImporter.class.getName());
```

Comment:
Does it mean we don't have a logger at all anymore in this class? It would have been sufficient to change the import and also to remove `.getName()` when creating the logger if we want to move to log4j.

Reply:
It is not needed in this class anymore as there is no logging done. It was just initialized but not used anywhere so I removed it.